packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -412,8 +412,12 @@ export const createSeatManager = (
           }
         },
         reallocate(/** @type {ZCFSeat[]} */ ...seats) {
-          seats.forEach(assertActive);
-          seats.forEach(assertStagedAllocation);
+          for (const seat of seats) {
+            assertActive(seat);
+          }
+          for (const seat of seats) {
+            assertStagedAllocation(seat);
+          }
```

Comment:
@Chris-Hibbert  would you consider it acceptable to have these two loops combined into one?

Reply:
It's okay. The only side effect is to throw. It could change what violation gets thrown first but nothing should depend on that.

Reply:
I agree, it would be fine to merge them.

Reply:
We still have staged allocations? I thought we got rid of those a long time ago. Any remaining uses?

Reply:
> I thought we got rid of those a long time ago.

We got rid of them from our code a long time ago, but we're still working through a process of removing them from the API. Upgrade 15 only had `atomicRearrange()` as a helper, the first appearance of it as a ZCF method on mainnet is in U16, which is imminent. Once that's out there, I'll feel comfortable about removing the vestiges of `reallocate()`